[DOC-898] Docs for Voyager v2025.8.1 (#28142)
yugabyte/yugabyte-db: docs/content/preview/yugabyte-voyager/reference/bulk-data-load/import-data-file.md

| Truncate tables on target YugabyteDB database before importing data. This option is only valid if `--start-clean` is set to true. <br>Default: false |

| --error-policy |

```yaml{.nocopy}
import-data-file:
  error-policy:
```

> | Defines the desired behavior when an error occurs while processing and importing rows to target YugabyteDB. Errors can arise from reading data from file, transforming rows, or ingesting them into YugabyteDB. <br> Accepted parameters: <ul><li> `abort` (Default): Immediately aborts the process.</li><li> `stash-and-continue` : Stashes the errored rows to a file and continues the import.</li></ul> |

Reviewer: ```suggestion
| Specifies how to handle errors when processing and importing rows to the target YugabyteDB database. Errors can arise from reading data from file, transforming rows, or ingesting them into YugabyteDB. <br> Accepted parameters: <ul><li> `abort` (Default) - Immediately aborts the process.</li><li> `stash-and-continue` - Stashes the errored rows to a file and continues the import.</li></ul> |
```
